.github/workflows/run-onert-micro-unit-tests.yml

```diff
@@ -0,0 +1,54 @@
+name: Run onert-micro Unit tests
+
+on:
+  pull_request:
+    branches:
+      - master
+      - release/*
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  run-onert-micro-unit-tests:
+    name: Run onert-micro Unit tests
+    runs-on: ubuntu-20.04
+    # Skip on draft, check on draft -> ready
+    if: github.event.pull_request.draft == false
+
+    steps:
+      - uses: dorny/paths-filter@v2
```

Comment:
@SlavikMIPT Sorry. I have one additional question. Why don't you use built-in filter? 
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths

Reply:
Thanks - I will refactor this, I don't have much experience with github-actions